apps/aecore/src/aec_coinbase_tx.erl

```diff
@@ -72,18 +77,21 @@ signers(#coinbase_tx{account = AccountPubkey}) -> [AccountPubkey].
 -define(CB_TX_VSN, 1).
 
 -spec serialize(tx()) -> [map()].
-serialize(#coinbase_tx{account = Account}) ->
+serialize(#coinbase_tx{account = Account, block_height = Height}) ->
     [#{<<"vsn">> => version()},
-     #{<<"acct">> => Account}].
+     #{<<"acct">> => Account},
+     #{<<"h">> => Height}].
```

Comment:
This affects the txs Merkle tree root hash in the block header hence consensus. Please bump db_path in deploy script.